hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestOperationConifg.scala

```diff
@@ -0,0 +1,201 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.hudi
+
+import org.apache.spark.sql.catalyst.TableIdentifier
+import org.apache.spark.sql.catalyst.catalog.{HoodieCatalogTable, SessionCatalog}
+import org.apache.spark.sql.hudi.command.InsertIntoHoodieTableCommand.buildHoodieInsertConfig
+
+import scala.reflect.ClassTag
+
+abstract class TestOperationConfig extends HoodieSparkSqlTestBase {
+  val catalog: SessionCatalog = spark.sessionState.catalog
+
```

Comment:
No need to make the clazz abstract, if you wanna to define some common utilities methods, just declare a companion object class with the same name.

Reply:
This abstract class aims to be inherited to test other operation like `buildHoodieDeleteTableConfig`, `buildHoodieDropPartitionsConfig` etc, a companion object is more reasonable if we do not need test other operation.